js/libs/keycloak-admin-client/src/defs/configPropertyRepresentation.ts

```diff
@@ -10,4 +10,5 @@ export interface ConfigPropertyRepresentation {
   options?: string[];
   secret?: boolean;
   required?: boolean;
+  stringify?: boolean;
```

Comment:
Properties in representations of the admin client should only be added if the fields actually exist on the representation on the Java-side, which this does not.

Reply:
The core issue here is, that all ConfigProperties of ClientMappers must be strings and therefore should be stringified. I have multiple solutions in mind, what is you opinion on that?

- Hardcode the stringify attribute in the frontend, even though it does not exist in the interface in the backend. Because some elements use it, it should be part of the configPropertyRepresentation. 
- Extend the representation in the frontend and backend with the stringify attribute, to support setting it manuelly in the mappers
- Change the expected request object on the backend side from Map<String,String> to Map<String,Object>

From my current understanding, solution two would be the appropriate solution, right?

Reply:
Yeah this is a hard one, I think ideally the backend determines which fields should be stringified, but I don't believe this is accurately done or consistently for that manner.

Reply:
I added it to the backend as well, so there is at least the possibilty to define and use it